docs/rules/arrow-parens.md

```diff
@@ -177,6 +178,7 @@ a.then(foo => { if (true) {} });
 (a = 10) => a;
 ([a, b]) => a;
 ({a, b}) => a;
+const f = (/** @type {number} */a) => a + a;
```

Comment:
:+1: 

It might be nice to also add a couple of `correct` examples with random comments, to show that we're not checking jsdoc, e.g. these two:

```js
const g = (/* comment */ a) => a + a;
const h = (a /* comment */) => a + a;
```